src/Microsoft.Win32.Registry/tests/Microsoft.Win32.Registry.Tests.csproj

```diff
@@ -56,6 +59,11 @@
     <Compile Include="TestData.cs" />
     <Compile Include="XunitAssemblyAttributes.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='net46'">
```

Comment:
One issue with building this way is that you aren't actually compiling the test library against the contract which might mean there are differences which might show up but be masked.